test/test_run.py

```diff
@@ -246,13 +245,15 @@ def run_it():
                   if fn != 'machine.json']
 
     data = util.load_json(result_fn)
-    value = dict(zip(data['result_columns'], data['results']['time_examples.TimeSuite.time_example_benchmark_1']))
+    value = dict(zip(data['result_columns'], data['results']
+                 ['time_examples.TimeSuite.time_example_benchmark_1']))
```

Comment:
This is way shorter than 99 and doesn't split the dictionary access among lines.
```suggestion
    value = dict(zip(
        data['result_columns'],
        data['results']['time_examples.TimeSuite.time_example_benchmark_1']))
```